packages/cactus-plugin-ledger-connector-quorum/src/main/json/openapi.json

```diff
@@ -710,6 +693,75 @@
                     }
                 }
             }
+        },
+        "/api/v1/plugins/@hyperledger/cactus-plugin-consortium-manual/consortium/jws": {
```

Comment:
@petermetz I had to add these endpoints to Quorum openapi document because test api-client-routing-node-to-node.test.ts from cactus-test-api-client didn't working. I think it was trying to call consortium-manual endpoints through Quorum API client and validation was failing.
Is there any way to do it better?

Reply:
@elenaizaguirre It's most likely being caused by the open API validator trying to look up the validation metadata of the consortium manual plugin and not finding it, showing screenshots below with how I debugged into it.
One more tip: If you want to see what request handlers are registered in an express app you can run `app._router.stack` on the debug console to print it like I did (see other screenshot)

TLDR solution: you might need to tie back together the consortium manual plugin PR and the quorum connector PR because of this dependency.

![Screenshot 2021-09-01 141141](https://user-images.githubusercontent.com/1583231/131746304-ea62926d-8979-456d-8e99-9bc6c2aef4a1.png)
![Screenshot 2021-09-01 140852](https://user-images.githubusercontent.com/1583231/131746307-52ee88cb-1799-4893-9859-d5da645aafa3.png)
![Screenshot 2021-09-01 140009](https://user-images.githubusercontent.com/1583231/131746309-43b03164-271d-4726-9e55-e4466dc2cfd0.png)